src/pages/api/v1/ph.js

```diff
@@ -0,0 +1,37 @@
+import mysql from 'mysql2/promise';
+
+export default async function handler(req, res) {
+    // Only allow GET-requests
+    if (req.method !== "GET") {
+        console.log(`Error: Method ${req.method} not allowed.`)
+        res.status(405)        // 405: method not allowed
```

Comment:
If you want to explain what a specifc number represents then create a const `NOT_ALLOWED` to improve readability. It is a bad smell if a comment has to explain it.

`res.status(NOT_ALLOWED)`